blocks/trending-news/trending-news.css

```diff
@@ -0,0 +1,181 @@
+.block.trending-news .link-color {
+    font-size: var(--body-font-size-xxl);
+    line-height: 1.5rem;
+    font-weight: var(--font-weight-bold);
+    color: var(--link-color-orange);
+    text-decoration: none;
+}
+
+.block.trending-news .container .title {
+    margin-bottom: 2rem;
+    text-align: center;
+}
+
+.block.trending-news .container .title h2 {
+    font-size: var(--heading-font-size-xmxl);
+    color: var(--primary);
+    line-height: 2.813rem;
+    text-transform: uppercase;
+    font-weight: var(--font-weight-bold);
+}
+
+.block.trending-news .news-slider .news-track {
+    padding-left: 1rem;
+    position: relative;
+    left: 0;
+    top: 0;
+    display: flex;
+    transition: transform 0.5s ease;
+    min-height: 17.5rem;
+    transform: translate3d(0, 0, 0);
+}
+
+.block.trending-news .news-slider .news-track > * {
+    opacity: 0;
+ }
+
+ .block.trending-news .news-slider .news-track > *:first-child {
+    opacity: 1;
+ }
+
+.block.trending-news .news-slider .news-card {
+    width: 25rem;
+    height: 20rem;
+    flex: 0 0 auto;
+    min-height: var(--space-1px);
+    transition: opacity 0.1s ease;
+}
+
+.block.trending-news .news-section .article {
+    background: transparent;
+    position: relative;
+    border: var(--space-1px) solid var(--boarder-trendign-news-card);
+    height: 100%;
+}
+
+.block.trending-news .news-section .article .picture-wrapper img {
+    object-fit: initial;
+    width: 23.375rem;
+    height: 10rem;
+    display: block;
+    transition: 0.3s;
+}
+
+.block.trending-news .news-section .article .article-text {
+    padding: var(--space-10px) 0;
+}
+
+.block.trending-news .news-section .article .article-text a {
+    padding-right: 2rem;
+    margin-bottom: 1.125rem;
+    font-size: var(--space-15px);
+    line-height: 1.375rem;
```

Comment:
@ravverma https://github.com/aemsites/icicidirect/blob/main/styles/styles.css#L142 constant

Reply:
it is greater then 1